src/leiningen/new/re_frame/project.clj

```diff
@@ -17,14 +19,19 @@
                                     "test/js"{{/test?}} {{#garden?}}
                                     "resources/public/css/compiled"{{/garden?}}]
 
```

Comment:
Oh nice, forgot about css - maybe rewrite this section so the css-dirs is always available

```
 :figwheel {:css-dirs ["resources/public/css"] {{#handler?}}
            :ring-handler {{name}}.handler/handler{{/handler?}} }
```